mkosi/__init__.py

```diff
@@ -2241,6 +2242,56 @@ def install_openmandriva(args: CommandLineArguments, root: str, do_run_build_scr
     disable_pam_securetty(root)
 
 
+@complete_step("Installing Gentoo")
+def install_gentoo(args: CommandLineArguments, root: str, do_run_build_script: bool) -> None:
+    try:
+        import portage
+    except ImportError:
+        die("you need portage module for Gentoo: https://gitweb.gentoo.org/proj/portage.git")
+
+    from _emerge.actions import load_emerge_config
+    from _emerge.actions import action_info
+    from _emerge.actions import action_sync
+    from _emerge.actions import action_build
+
+    portdir = "/var/db/repos/gentoo"
+    pkgdir = "/var/cache/binpkgs"
+    os.makedirs(portdir, exist_ok=1)
+    os.makedirs(pkgdir, exist_ok=1)
```

Comment:
Please use booleans, this is not C :)
```suggestion
    os.makedirs(portdir, exist_ok=True)
    os.makedirs(pkgdir, exist_ok=True)
```

Reply:
noted